pkg/diff/diff_test.go

```diff
@@ -163,7 +163,6 @@ F`
 		// We do need to cleanup, as otherwise we get some spurious changes on complex diffs
 		diffs = dmp.DiffCleanupSemantic(diffs)
 
-		t.Logf("diffs %v", diffs)
```

Comment:
Ah - actually - this one is so that we can see the actual diff in the case that the test fails.

We choose to then set `-v`, which is why the results appear, but that is correct.

Reply:
shouldn't we be only outputting the log if we fail??

Reply:
I thought that's how `go test` worked by default.  But let's get this merged..